### ISSUE TITLE
How to stay up to date?

### ISSUE THREAD

**Issue by timkgh** [Lochnair/vyatta-wireguard#82]
What is the process of staying up to date? In particular when there are security fixes.
Is there a way to be notified of new releases without subscribing to the entire github repo? Something like a mailing list?
Or if there's an automated process on the router to check and notify for updates, that would be great.

**Comment by storrgie** [comment]
You can add the [releases](https://github.com/Lochnair/vyatta-wireguard/releases) page to a feed reader.

**Comment by timkgh** [comment]
> Apparently GitHub has recently added a new notifications option called `Releases only` for this exact purpose.

Wow finally they support it! Been waiting for it for years.

**Comment by timkgh** [comment]
Follow-up: once there is a new release what is the update process on an ER-X so that existing configuration is preserved?

**Comment by whiskerz007** [comment]
> There is a new firmware update for the ER routers and once applied it seems to have wiped Wireguard. Is it not supposed to survive firmware upgrades and has to be installed again after?

TL;DR - EdgeOS does not preserve packages through firmware upgrades. ([link](https://help.ubnt.com/hc/en-us/articles/205202560-EdgeRouter-Add-Debian-Packages-to-EdgeOS))

@bits01 please refer to [this](https://github.com/Lochnair/vyatta-wireguard/issues/62) post. If you haven't made any changes in the web-ui, or saved any changes from the commandline-ui, you can use the following commands to rebuild the config to what it was prior to the upgrade ([reference](https://github.com/Lochnair/vyatta-wireguard/issues/72#issuecomment-423840448)).

```
configure
sudo dpkg -i /PATH/TO/PACKAGE.DEB
sudo modprobe wireguard
load
commit
exit
```

**Comment by whiskerz007** [comment]
I've developed a script that will install/update WireGuard, without the need of a reboot, and preserve the package installation after a firmware upgrade. ([link](https://github.com/whiskerz007/ubnt_get_wireguard))

**Comment by dcormier** [comment]
> I've developed a script that will install/update WireGuard, without the need of a reboot, and preserve the package installation after a firmware upgrade. ([link](https://github.com/whiskerz007/ubnt_get_wireguard))

It would be really nice to see this integrated into this repo.
